### ISSUE TITLE
Работает ли bem-tools под windows ?

### ISSUE THREAD

**Issue by rtxrulez** [bem/bem-forum-content-ru#258]
И если он не работает по виндой. То тогда как же верстальщики верстают сайты из под линукса? Там же нет фотошопа ? Или как в яндексе устроено рабочее окружение верстальщика... Мне например мешает перейти на линукс только отсутствие фотошопа и ie для тестов.

**Comment by kuflash** [comment]
@rtxrulez, bem-tools и enb прекрасно работают под виндой. Ну а на сколько я знаю, в яндексе большинство юзает MacOS, а там, как известно фотошоп есть, ну а для тестов на IE много всего сейчас есть, от виртуалок, до modern.ie

**Comment by Guria** [comment]
Да, всё должно работать под виндой, но я предпочитаю использовать такие вещи в linux окружении. Работая в винде, я обычно всё поднимаю в виртуалках.

**Comment by tadatuta** [comment]
@rtxrulez При установке `git`, ставится git bash, где эмулируется unix-окружение. Этого достаточно для работы `bem-tools` и `ENB`.

**Comment by rtxrulez** [comment]
Для работы может быть достаточно. Вот только git bash не такой удобный как на линуксе.  Может в яндексе есть люди которые занимаются версткой из под линукс системы ? Как они работают с фотошопом и ie ? Какие особенности ?

**Comment by kuflash** [comment]
@rtxrulez, я в качестве терминала на винде юзаю это: http://bliker.github.io/cmder/ Вполне очень удобно.

**Comment by tadatuta** [comment]
@rtxrulez все, для кого фотошоп актуален, сидят на маках. но если хочется именно родную юниксовую консоль, то порекомендую винду + вагрант с виртуальной убунтой.

**Comment by dab** [comment]
поддержу @tadatuta, когда разрабатывал на винде — использовал VirtualBox как виртуальный сервер для разработки. Обращаться к нему для редактирования можно по ssh, поднять на нем окружение правильное и тестовый  linux-сервер под рукой всегда.

**Comment by belozyorcev** [comment]
@Guria я в качестве основной системы использую ubuntu. А фотошоп установлен на виртуальной машине (WinXP).
Не могу я в винде работать... Неудобно.

Хотя 2-й системой стоит Win8.1 (а точнее уже Win10TP), использую в крайних случаях.

**Comment by kompolom** [comment]
Еще есть extract от adobe который частично может заменить фотошоп. Сам фотошоп, правда не первой свежести, запускается через wine.
